apps/ensrainbow/src/lib/api.ts

```diff
@@ -29,7 +30,11 @@ export async function createApi(db: ENSRainbowDB): Promise<Hono> {
     logger.debug(`Healing request for labelhash: ${labelhash}`);
     const result = await server.heal(labelhash);
     logger.debug(`Heal result:`, result);
-    return c.json(result, result.errorCode);
+
+    // Map error codes > 1000 to 500, otherwise use the original code
+    const statusCode = result.errorCode && result.errorCode >= 1000 ? 500 : result.errorCode;
```

Comment:
Can `result.errorCode >= 100` be ever `true`? We haven't changed anything int the server code, so it won't know about new error codes.

Reply:
There are new error codes defined:
```
  // Network error codes
  TIMEOUT: 1000,
  NETWORK_OFFLINE: 1001,
  GENERAL_NETWORK_ERROR: 1099,
  ```